libraries/botbuilder-dialogs-adaptive/src/input/inputDialog.ts

```diff
@@ -382,16 +382,31 @@ export abstract class InputDialog extends Dialog implements InputDialogConfigura
             msg.inputHint = InputHints.ExpectingInput;
         }
 
+        this.trackGeneratorResultEvent(dc, template, msg);
+
+        return msg;
+    }
+
+    /**
+     * @protected
+     * Track GeneratorResultEvent telemetry event with InputDialogResultEvent context.
+     * @param dc The [DialogContext](xref:botbuilder-dialogs.DialogContext) for the current turn of conversation.
+     * @param activityTemplate used to create the Activity.
+     * @param msg The [IMessageActivity](xref:botframework-schema.IMessageActivity) which will be sent.
```

Comment:
Should this comment be something regarding Partial [Activity](xref:botframework-schema.Activity) type, not IMessageActivity?